posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta"
```

Comment:
Java 17 beta support, JWE support for access and ID tokens, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta